src/sentry/static/sentry/app/components/charts/miniBarChart.tsx

```diff
@@ -142,7 +151,7 @@ class MiniBarChart extends React.Component<Props> {
           }),
         },
       };
-      series.push(markerSeries);
+      series?.push(markerSeries);
```

Comment:
Is this optional chain needed? `series` should always be an array based on 77 - 101

Reply:
No, it was a leftover from some code I scrapped. 👍